### DISCUSSION TITLE
Grief Prevention on Forge 1.19.3

### DISCUSSION THREAD

**Discussion by OhNoScorpio** [GriefPrevention/GriefPrevention#2024] [General]
Hi friends,

I feel a bit silly asking this but I can't make the mod show up in my game. I run a modded server on Forge 1.19.3, I got the latest version of this mod and put it in my own and the server's mod folder, but nothing shows up in game. I've seen guides for Bukkit servers, but I can't seem to make it work on Forge. Am I missing something?

Thanks! xx

**Comment by Jikoo** [comment]
This is a Bukkit plugin, not a Forge mod. There was an unofficial Forge port a couple years back, but it was discontinued.

As much as there are Forge+Bukkit hybrid servers, they have historically been very inconsistent due to the ways mods work, and GP no longer has modded block support at all. I strongly recommend you run a protection system native to the platform you're using instead.

**Reply by OhNoScorpio** [reply]
aw thank you, I was afraid this was the case.. you don't happen to have a suggestion? It should ideally be something very intuitive as most server players are new/inexperienced.

**Reply by Jikoo** [reply]
[FTB-Chunks](https://github.com/FTBTeam/FTB-Chunks), perhaps? I'm afraid I'm not a Forge modder so I'm pretty out of touch with that side of the Minecraft scene. Downside is that it doesn't let you do things like accrue chunks over play but you could probably supplement that with permissions for increased chunk counts and an advancement track. Also not sure if there are restrictions for adding it to a non-FTB pack, they don't seem to have posted it with a license. At the very least, Forge users are likely familiar with it.
